migration/transitioning.rst

```diff
@@ -384,9 +391,233 @@ currently logged in customer - even though this is complex analytical
+
+.. code-block:: python
+
+  from __future__ import unicode_literals
+  from django.db import migrations
+
+  class Migration(migrations.Migration):
+    dependencies = [
+      ('appname', '<name of latest migration>')
+    ]
+
+    operations = [
+      # Django considers "id" the primary key of these tables, but
+      # the database mustn't, because the primary key will be composite
+      migrations.RunSQL(
+        "ALTER TABLE mtdjango_product DROP CONSTRAINT mtdjango_product_pkey;",
+        "ALTER TABLE mtdjango_product ADD CONSTRAINT mtdjango_product_pkey PRIMARY KEY (id)"
+      ),
+      migrations.RunSQL(
+        "ALTER TABLE mtdjango_purchase DROP CONSTRAINT mtdjango_purchase_pkey;",
+        "ALTER TABLE mtdjango_purchase ADD CONSTRAINT mtdjango_purchase_pkey PRIMARY KEY (id)"
+      ),
+    ]
+
+Next, we'll make one to tell Citus to mark tables for distribution. :code:`./manage.py makemigrations citus --empty --name distribute_tables`. Edit the result to look like this:
+
+.. code-block:: python
+
+  from __future__ import unicode_literals
+  from django.db import migrations
+
+  class Migration(migrations.Migration):
+    dependencies = [
+      # leave this as it was generated
+    ]
+
+    operations = [
+      migrations.RunSQL(
+        "SELECT create_distributed_table('mtdjango_store','id')"
+      ),
+      migrations.RunSQL(
+        "SELECT create_distributed_table('mtdjango_product','store_id')"
+      ),
+      migrations.RunSQL(
+        "SELECT create_distributed_table('mtdjango_purchase','store_id')"
+      ),
+    ]
+
+Finally, we'll establish a composite foreign key. :code:`./manage.py makemigrations citus --empty --name composite_fk`.
+
+.. code-block:: python
+
+  from __future__ import unicode_literals
+  from django.db import migrations
+
+  class Migration(migrations.Migration):
+    dependencies = [
+      # leave this as it was generated
+    ]
+
+    operations = [
+      migrations.RunSQL(
+        """
+            ALTER TABLE mtdjango_purchase
+            ADD CONSTRAINT mtdjango_purchase_product_fk
+            FOREIGN KEY (store_id, product_id)
+            REFERENCES mtdjango_product (store_id, id)
+            ON DELETE CASCADE;
+        """,
+        "ALTER TABLE mtdjango_purchase DROP CONSTRAINT mtdjango_purchase_product_fk"
+      ),
+    ]
+
+Apply the migrations by running :code:`./manage.py migrate`.
+
+At this point the Django application models are ready to work with a Citus backend. You can continue by importing data to the new system and modifying controllers as necessary to deal with the model changes.
+
+Updating the Django Application
+*******************************
+
+To simplify queries in the Django application, Citus has developed a Python library called `django-multitenant <https://github.com/citusdata/django-multitenant>`_. Include :code:`django-multitenant` in the :code:`requirements.txt` package file for your project, and then modify your models.
```

Comment:
Just mention that this library is in **Beta** because it has to be still tested well.